hreuniversalis/missions/VN_zz_Prussian_missions.txt

```diff
@@ -888,7 +888,20 @@
 # 				name = "pru_unified_germany_mission"
 # 				duration = -1
 # 			}
-#         }
+#
+#			country_event_with_effect_insight = {
+#				id = flavor_ger.11
+#				effect = "
+#					custom_tooltip = GER_unlock_reform
+#					
+#					GER_unlock_reform_based_on_culture = { reform = yes }
+#					
+#					custom_tooltip = \" \"
+#					
+#					custom_tooltip = GER_we_may_choose_not_to_take_it
+#				"
+#			}
+#       }
```

Comment:
Not quite sure what you want to do here: in the base game, it seems the `JHC_GER_kaiserreich` mission is what allows you to break the cultural fragmentation reform, and gives you the choice of a reform based upon your primary culture or just a governing capacity one.

I've heard you dislike missions and wanted to remove them, so I'm guessing that's why these are all commented out (I might be a few versions behind as I do some modifications myself and therefore manage updates myself), but no missions => eternally locked t2 reform I think.

Multitude of other ways to address it I suppose though.

Reply:
Ended up reverting this, adding a commented-out mission reword that doesn't perfectly map into this mod isn't great.